R/compile-dll.R

```diff
@@ -13,16 +13,23 @@
 #'   `PKG_CPPFLAGS=`$(R_HOME)/bin/Rscript -e 'Rcpp:::CxxFlags()'``
 #'
 #' @inheritParams build
+#' @param force If `TRUE`, for compilation even if [needs_compile()] is
+#'   `FALSE`.
 #' @seealso [clean_dll()] to delete the compiled files.
 #' @export
-compile_dll <- function(path = ".", quiet = FALSE) {
+compile_dll <- function(path = ".",
+                        force = FALSE,
+                        compile_attributes = pkg_links_to_rcpp(path),
```

Comment:
`compile_dll()` on the other hand is usually used interactively, so there I think it does make sense to compile attributes by default (assuming it uses Rcpp) and if not to register the routines.